biosimulations/apps/dispatch-api/src/app/app.service.ts

```diff
@@ -85,6 +85,12 @@ export class AppService {
     };
   }
 
+  private async getJobCancel(uuid: string) {
```

Comment:
Is this designed to cancel a running job?

Reply:
Yes, but it's still incomplete. Not properly tested